composer.json

```diff
@@ -3,11 +3,15 @@
     "description": "RabbitMQ and PHP client for Laravel that allows you to add and listen queues messages just simple",
     "keywords": ["laravel5", "laravel", "rabbitmq", "queue", "package", "client", "Victor Cruz", "mookofe" ],
     "license": "MIT",
-    "version": "v1.0.4",
+    "version": "v1.0.5",
     "authors": [
         {
             "name": "Victor Cruz",
             "email": "cruzrosario@gmail.com"
+        },
+        {
+            "name": "Martin Hilscher",
+            "email": "hilscher@jungehaie.com"
         }
```

Comment:
Please remove this reference as author. In this case you will become a contributor not an author. Your contributions will show up here: https://github.com/mookofe/tail/graphs/contributors